<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mwListe</class>
 <widget class="QMainWindow" name="mwListe">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/res.qrc">
    <normaloff>:/icons/button/edit.svg</normaloff>:/icons/button/edit.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="widgetListe" name="listM">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>473</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actFind"/>
    <addaction name="actExHTML"/>
    <addaction name="separator"/>
    <addaction name="actOptions"/>
    <addaction name="separator"/>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;Affichage</string>
    </property>
    <addaction name="actTypeFilms"/>
    <addaction name="actTypeZik"/>
    <addaction name="actTypeBook"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actOptions">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/icons/button/misc.svg</normaloff>:/icons/button/misc.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/icons/button/finish.svg</normaloff>:/icons/button/finish.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
  </action>
  <action name="actExHTML">
   <property name="icon">
    <iconset resource="../res/res.qrc">
     <normaloff>:/icons/button/www.svg</normaloff>:/icons/button/www.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exporter</string>
   </property>
  </action>
  <action name="actTypeFilms">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Films</string>
   </property>
  </action>
  <action name="actTypeZik">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Musique</string>
   </property>
  </action>
  <action name="actTypeBook">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Livres</string>
   </property>
  </action>
  <action name="actFind">
   <property name="text">
    <string>&amp;Rechercher</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>widgetListe</class>
   <extends>QListWidget</extends>
   <header>src/widgetListe.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>mwListe</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
